edge/pkg/common/util/config.go

```diff
@@ -78,7 +77,19 @@ func GenerateTestYaml(test interface{}, path, filename string) error {
 	if err != nil {
 		return err
 	}
-	err = ioutil.WriteFile(path+"/"+filename+".yaml", data, 0777)
+	fp, err := os.Create(path + "/" + filename + ".yaml")
+	if err != nil {
+		return err
+	}
+	_, err = fp.Write(data)
+	if err != nil {
+		return err
+	}
+	err = fp.Sync()
```

Comment:
What about:
```
if err := fp.Sync(); err != nil {
  return err
}
```

for the sake of saving one line?

Reply:
Done, thanks